src/layout/Header/MenuIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { IconMenu } from './styled'
+
+interface Props {
+  active: boolean
+  handleMobileMenu: () => void
+}
+
+export const MenuIcon: React.FC<Props> = ({ active, handleMobileMenu }) => {
```

Comment:
when click on menu button, its moving from one position to other position

https://user-images.githubusercontent.com/6604146/176940982-3754b473-de4c-4e20-9172-47f29238ede7.mov

Reply:
This is strange, this doesn't happen to me

https://user-images.githubusercontent.com/37405304/177000872-59e53785-b5ac-4954-bfc5-29ab5c2d5095.mp4

Reply:
In mobile work normally

https://user-images.githubusercontent.com/37405304/177000961-c9af8994-fe3a-47a0-b1fb-8fa0c079c30c.mp4

Reply:
looks good in mobile. lets skip this for now.